core-application/src/main/java/gohigher/application/port/out/persistence/ApplicationPersistenceQueryPort.java

```diff
@@ -9,5 +10,7 @@ public interface ApplicationPersistenceQueryPort {
 
 	Optional<Application> findById(Long id);
 
-	List<Application> findByIdAndMonth(Long userId, int year, int month);
+	List<Application> findByUserIdAndMonth(Long userId, int year, int month);
+
+	List<Application> findByUserIdAndDate(long userId, LocalDate date);
```

Comment:
```suggestion
	List<Application> findByUserIdAndDate(Long userId, LocalDate date);
```
Request 에서는 Wrapper 로 받아왔는데, 별다른 검증 없이 long 으로 받는다면 NPE 가 발생할 수도 있을것 같습니다.

Reply:
맞는 말씀이십니다! 반영하겠습니다 :)